setup.cfg

```diff
@@ -37,7 +37,9 @@ packages = find:
 include_package_data = True
 
 install_requires =
-  frozenlist >= 1.1.0
+  frozenlist >= 1.1.0,
+  typing_extensions; python_version == '3.9'
```

Comment:
What's the minimum version of the dependency implementing `ParamSpec`? We should set the lower limit to that.

Additionally, please use `-` instead of `_` in the PyPI project name on this line.